web/api/v2/public/apps/index.ts

```diff
@@ -14,6 +14,14 @@ import {
 } from "./graphql/get-app-rankings.generated";
 import { getSdk as getHighlightsSdk } from "./graphql/get-app-web-highlights.generated";
 
+export type GetAppsResponse = {
+  app_rankings: {
```

Comment:
I think you can import the response type of this directly from the graphql generated file